backend/src/main/resources/db-migrations/common/v0-[common]-schema-init.sql

```diff
@@ -59,11 +59,32 @@ CREATE INDEX `ix_alert_type_code` on `alert_type` (`code`);
 
 
 
+CREATE TABLE `language` (
+	`id` VARCHAR(64) NOT NULL,
+
+	`code` VARCHAR(16) NOT NULL,
+	`description` VARCHAR(256),
+	`iso_code` VARCHAR(16),
+	`ms_locale_code` VARCHAR(16),
+
+	-- audit fields
+	`created_by` VARCHAR(64) NOT NULL,
+	`created_date` TIMESTAMP WITH TIME ZONE NOT NULL,
+	`last_modified_by` VARCHAR(64),
+	`last_modified_date` TIMESTAMP WITH TIME ZONE,
+
+	CONSTRAINT `pk_language` PRIMARY KEY (`id`)
+);
+
+CREATE INDEX `ix_language_code` on `language` (`code`);
```

Comment:
Since we can (presumably) look up a language via `code`, `iso_code`, and `ms_locale_code`, we should create indexes for them, I think.

Reply:
added